corehq/apps/data_vault/models.py

```diff
@@ -0,0 +1,15 @@
+import secrets
+
+from django.db import models
+
+
+class VaultStore(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    key = models.CharField(max_length=25, unique=True)
```

Comment:
specifying unique here will also create a `varchar_pattern_ops` index as well. You can avoid the extra index by specifying the index in the class Meta: https://stackoverflow.com/questions/32773049/how-to-remove-index-varchar-pattern-ops-in-a-django-1-8-migration/50926644#50926644

Reply:
Oh, That is interesting,
https://github.com/dimagi/commcare-hq/pull/28161/commits/15796a4873853349f001c59045cd74b43ddd9109